server/server.js

```diff
@@ -1,7 +1,7 @@
 /* eslint no-console:0 */
 /* globals console */
 import express from 'express';
-import mongoose from 'mongoose';
+// import mongoose from 'mongoose';
```

Comment:
Probably not for now. :+1: to at least keeping it commented out until/if we decide to use it. I could see using it for storing room credentials, as an alternative to env credentials.